sklearn/cluster/birch.py

```diff
@@ -74,7 +74,7 @@ def _split_node(node, threshold, branching_factor):
 
     farthest_idx = np.unravel_index(
         dist.argmax(), (n_clusters, n_clusters))
-    node1_dist, node2_dist = dist[[farthest_idx]]
+    node1_dist, node2_dist = dist[tuple(farthest_idx)]
```

Comment:
I'm not sure that this is the intention here. I think the list might have been intended to extract a result with a different shape.... Please double check